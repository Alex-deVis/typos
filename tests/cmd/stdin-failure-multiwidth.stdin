한 Apropriate world
한 Apropriate world
😵‍💫 Apropriate world
👁️‍🗨️ Apropriate world
	Apropriate world