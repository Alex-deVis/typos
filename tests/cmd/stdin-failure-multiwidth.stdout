error: `Apropriate` should be `Appropriate`
  --> -:5:3
  |
5 | 한 Apropriate world
  |    ^^^^^^^^^^
  |
error: `Apropriate` should be `Appropriate`
  --> -:12:3
   |
12 | 한 Apropriate world
   |    ^^^^^^^^^^
   |
error: `Apropriate` should be `Appropriate`
  --> -:19:3
   |
19 | 👁️‍🗨️ Apropriate world
   |    ^^^^^^^^^^
   |
error: `Apropriate` should be `Appropriate`
  --> -:26:3
   |
26 | 😵‍💫 Apropriate world
   |    ^^^^^^^^^^
   |
error: `Apropriate` should be `Appropriate`
  --> -:33:2
   |
33 |  Apropriate world
   |  ^^^^^^^^^^
   |
